pkg/monitor/dissect.go

```diff
@@ -232,6 +232,9 @@ func Dissect(dissect bool, data []byte) {
 		if parser.Truncated {
 			fmt.Println("  Packet has been truncated")
 		}
+		if err != nil {
+			fmt.Println("  Failed to decode layer:", err)
```

Comment:
nit: layers?

Reply:
My understanding is that `DecodeLayers decodes as many layers as possible from the given data`. If an error occurs, it would be on _one_ specific layer and would cause the decoding to stop. So there may be several layers that are not decoded, but the failure would only really happen on one layer?

I'm fine either way and happy to use the plural if you think it makes more sense.

Reply:
Nope, singular sounds good to me. Thanks!